app/ante/max_tx_size.go

```diff
@@ -18,11 +17,6 @@ func NewMaxTxSizeDecorator() MaxTxSizeDecorator {
 
 // AnteHandle implements the AnteHandler interface. It ensures that tx size is under application's configured threshold.
 func (d MaxTxSizeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (sdk.Context, error) {
-	// Tx size rule applies to app versions v3 and onwards.
-	if ctx.BlockHeader().Version.App < v3.Version {
-		return next(ctx, tx, simulate)
-	}
-
```

Comment:
unrelated removal, I believe we don't need this anymore with the multiplexer in place